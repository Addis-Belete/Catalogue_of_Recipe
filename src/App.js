import './App.css';

function App() {
  return (
    <div className="App">
      React Capstone Metahulish
    </div>
  );
}

export default App;
