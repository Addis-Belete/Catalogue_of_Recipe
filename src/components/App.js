import '../style/App.css';
import ItemList from '../container/ItemList';
import SearchBar from '../container/SearchBar';

const App = () => (
  <div className="App">
    <SearchBar />
    <ItemList />
  </div>
);

export default App;
