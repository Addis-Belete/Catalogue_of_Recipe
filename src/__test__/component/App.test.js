/* eslint-disable import/extensions */
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../components/App';

describe('App', () => {
  it('should render the catalogue of recipe', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
