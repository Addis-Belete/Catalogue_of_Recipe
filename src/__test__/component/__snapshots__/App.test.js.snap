// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the catalogue of recipe 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
     Catelogue of Recipe 
  </div>
  <div>
    <div
      class="category"
    >
      <input
        class="input"
        placeholder="Search for recipes"
        type="text"
        value=""
      />
      <select
        id="cat"
        name="category"
      >
        <option
          value="Beef"
        >
          Beef
        </option>
        <option
          value="Breakfast"
        >
          Breakfast
        </option>
        <option
          value="Chicken"
        >
          Chicken
        </option>
        <option
          value="Dessert"
        >
          Dessert
        </option>
        <option
          value="Goat"
        >
          Goat
        </option>
        <option
          value="Lamb"
        >
          Lamb
        </option>
        <option
          value="Miscellaneous"
        >
          Miscellaneous
        </option>
        <option
          value="Pasta"
        >
          Pasta
        </option>
        <option
          value="Pork"
        >
          Pork
        </option>
        <option
          value="Seafood"
        >
          Seafood
        </option>
        <option
          value="Side"
        >
          Side
        </option>
        <option
          value="Starter"
        >
          Starter
        </option>
        <option
          value="Vegan"
        >
          Vegan
        </option>
        <option
          value="Vegetarian"
        >
          Vegetarian
        </option>
      </select>
    </div>
    <div>
      <div>
        <div
          class="tablet desktop"
        >
          <div>
            <a
              href="/2020"
            >
              <img
                alt="recipeImage"
                class="img"
                src="undefined"
              />
              <p
                class="p"
              >
                hello world
              </p>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
